src/tests/activity.test.js

```diff
@@ -1 +1,141 @@
-test.skip('skip', () => {});
+import request from 'supertest';
+import app from '../app';
+import activity from '../models/activity';
+import activitySeed from '../seeds/activity';
+
+const mockActivity = {
+  name: 'Boxing',
+  description: 'vestibulum aliquet ultrices erat tortor sollicitudin mi non mi integer ac neque duis',
+  isActive: false,
+};
+
+const mockActivityExistingName = {
+  name: 'Chandal',
+  description: 'Any description',
+  isActive: false,
+};
+
+const mockLessName = {
+  name: '',
+  description: 'Some text',
+  isActive: true,
+};
+
+const mockLessDescription = {
+  name: 'Name',
+  description: '',
+  isActive: true,
+};
+
+const mockLessIsActive = {
+  name: 'Names',
+  description: 'some description',
+  isActive: '',
+};
+
+beforeAll(async () => {
+  await activity.collection.insertMany(activitySeed);
+});
+
+describe('GET /api/activities', () => {
+  test('The database must return 200 if is not empty of activities', async () => {
+    const response = await request(app).get('/api/activities').send();
+    expect(response.body.data.length).toBeGreaterThan(0);
+    expect(response.status).toBe(200);
+    expect(response.error).toBeFalsy();
+  });
+  test('The status must be 404, if the url is wrong', async () => {
+    const response = await request(app).get('/activities').send();
+    expect(response.status).toBe(404);
+    expect(response.error).toBeTruthy();
```

Comment:
We should access response.body.error
```suggestion
    expect(response.body.error).toBeTruthy();
```

Please apply this through all the file.

Reply:
doesn't pass the tests if i apply this changes

Reply:
In error 404 changed to toBeUndefined()